Scripts/Hooks/scratchpad-hook.lua

```diff
@@ -796,8 +831,22 @@ local function loadScratchpad()
                 addCoordinateListener = function(listener)
                     table.insert(coordListeners, listener)
                 end,
+                addFrameListener = function(listener)
+                    frameListeners[extid] = listener
+                end,
+                addmissionLoadEndListener = function(listener)
+                    table.insert(missionLoadEndListeners, listener)
+                end,
                 formatCoord = formatCoord,
-                log = log
+                log = log,
+                getSelection = getSelection,
+                getcurrentPage = function()
+                    return currentPage
+                end,
+                extid = extid,
+                setPageNotice = function(str)
+                    setPagesnotice(extid, str)
+                end,
```

Comment:
Would you mind documenting the plugin api changes [in the README](https://github.com/rkusa/dcs-scratchpad/blob/main/README.md?plain=1#L46-L72)?